src/structured-data/structured-data-tags.js

```diff
@@ -254,9 +254,33 @@ function generateLiveBlogPostingData(structuredData = {}, story = {}, publisherC
   };
 }
 
+function getEmbedUrl(cards) {
+  // get the first story element which has the embed url
+
+  let storyElemIndex;
+
+  const card = cards.find((card) => {
+    const storyElements = card["story-elements"];
+    return storyElements.find((elem, index) => {
+      if (elem["embed-url"]) {
+        storyElemIndex = index;
+        return elem["embed-url"];
+      }
+      return false;
+    });
+  });
+  if (card) {
+    const storyElement = card['story-elements'][storyElemIndex]
+    return storyElement["embed-url"];
+  }
+  return "";
+}
```

Comment:
Suggestion: 

```js
function getEmbedUrl(cards) {
  let jsEmbedUrl = "";
  cards.some((card) => {
    return card["story-elements"].find((elm) => {
      if (elm["embed-url"]) {
        jsEmbedUrl = elm["embed-url"];
        return true;
      }
      return false;
    });
  });
  return jsEmbedUrl;
}
```

Reply:
@ags1773 made some changes. find or some both are same.  Find is more appropriate here as we are finding something